### ISSUE TITLE
Just migrated to CodeWhisperer and it doesn't work

### ISSUE THREAD

**Issue by jvlobo** [withfig/fig#2938]
Hello!

Today when I opened Fig I saw the message that I had to migrate to CodeWhisperer so I did. 
The uninstalling of Fig and installing of CodeWhisperer was very promising but once the installation was done, nothing is working :(

I lost the autocompletion feature so I opened the CodeWhisperer app settings, there is a Help & Support section where it says to install the `Shell integrations`, so I tried to do that: click the button but nothing happens. The other option to `Enable accessibility` the same, button does nothing. The same if I click on the link `User Manual ->`
It seems to be an Electron app? so I can check the console and it is full of errors:

![Screenshot 2024-03-18 at 20 54 24](https://github.com/withfig/fig/assets/5671420/f80e9120-26eb-4b9d-807a-4c5009683942)

Any ideas on what is happening? I didn't do anything strange, just accepted on Fig.io to migrate to CodeWhisperer 🤔, nobody got this issue? This is my MacOS version: 14.2.1 (23C71)

Thanks!

**Comment by nathanaeng** [comment]
I'm having the exact same issue.

**Comment by wk13071526** [comment]
I'm having the exact same issue.

**Comment by NickRuigrok** [comment]
Same issue

**Comment by Gresham24** [comment]
Same problem here and finding no help at all. This is why I hate doing updates/upgrades immediately they happen

**Comment by jvlobo** [comment]
> Same problem here and finding no help at all. This is why I hate doing updates/upgrades immediately they happen

Yep... I hate it too, but I saw it and thought... well, it is from Amazon, at least it'll work no? big mistake :(

**Comment by grant0417** [comment]
This issue should be resolved, sorry about the issue. Update the app to `0.15.1` and you should be able to finish the migration.

<img width="170" alt="Screenshot 2024-03-19 at 2 46 58 PM" src="https://github.com/withfig/fig/assets/7355421/ef476fe2-5141-489e-b0e3-8bbf77ea1699">
